scvi/module/_mrdeconv.py

```diff
@@ -69,14 +70,15 @@ def __init__(
         self.n_latent = n_latent
         self.n_genes = n_genes
         self.amortization = amortization
+        self.l1_sparsity = l1_sparsity
         # unpack and copy parameters
         self.decoder = FCLayers(
             n_in=n_latent,
             n_out=n_hidden,
             n_cat_list=[n_labels],
             n_layers=n_layers,
             n_hidden=n_hidden,
-            dropout_rate=0,
+            dropout_rate=0.05,
```

Comment:
the zero here was actually super important. If you have layer norm you don't need dropout. And both batchnorm and dropout behaved wierdly for transfer learning so I had deactivated them (on this specific decoder). same for condscvi. Does this work? It's a bit dangerous because it behaves abnormally during .train() compared to .eval(). So I'd prefer to keep this zero unless it's motivated.

Reply:
It's essential to get comparable results with the old code after fixing the library size. It's working with both amortization schemes.

Reply:
Works perfectly fine on simulation (there is a gain of 5-10% on gene correlation figure from the paper with the updated code though I wasn't looking what change is responsible for this).